### ISSUE TITLE
docs.sandstorm.io sidebar is not collapsible

### ISSUE THREAD

**Issue by zarvox** [sandstorm-io/sandstorm#590]
Dunno if this is an issue with our code or a limitation of the mkdocs readthedocs theme, or what, but it makes the docs look a bit cluttered.

**Comment by paulproteus** [comment]
​Thanks for filing this!

https://trafficserver.readthedocs.org/en/latest/getting-started.en.html is
an example of what the expand/collapse looks like -- notice the icons next
to "Introduction" "Installation" etc.

**Comment by paulproteus** [comment]
Just to provide even more info:

* On https://docs.sandstorm.io/en/latest/ , in the left nav, I see:

```
Home
Sandstorm documentation
What is Sandstorm? See sandstorm.io
Choose your adventure
Using
```

* That's because the middle 3 items are section headings from the `Home` document.

* By contrast, on https://trafficserver.readthedocs.org/en/latest/getting-started.en.html , the section headings from that document are _not_ visible until you click the `+` on the left.

We'd like that behavior.

**Comment by paulproteus** [comment]
Hey @d0ugal do you have any idea if what we want is possible? And/or should I try to fix the `mkdocs` readthedocs.org theme?

(I just realized rather than Tweeting you a link to this, I can notify you in this issue!)

**Comment by paulproteus** [comment]
I now see that https://github.com/mkdocs/mkdocs/issues/588 relates to this!

**Comment by d0ugal** [comment]
Yup, that's correct - https://github.com/mkdocs/mkdocs/issues/588 should solve this. Help is definitely welcome, the task of trying to get the new CSS etc. to work well with our version is a tedious one.
